cpp/ql/test/query-tests/Likely Bugs/Format/WrongTypeFormatArguments/Buildless/tests.c

```diff
@@ -0,0 +1,11 @@
+// semmle-extractor-options: --expect_errors
+
+int printf(const char * format, ...);
+int fprintf();
+
+void f() {
+    printf("%s", 1); // BAD
+    printf("%s", implicit_function()); // GOOD - we should ignore the type
+    sprintf(0, "%s", ""); // GOOD
+    fprintf(0, "%s", ""); // GOOD
```

Comment:
This one is now good, because `int fprintf()` now no longer yields a value for `getFirstFormatArgumentIndex`?

Reply:
Yes.